third_party/forked/shell2junit/sh2ju.sh

```diff
@@ -24,17 +24,17 @@
 ###
 
 asserts=00; errors=0; total=0; content=""
-date=`which gdate 2>/dev/null || which date`
+date="$(command -v gdate 2>/dev/null || command -v date)"
```

Comment:
For reference - [SC2230](https://github.com/koalaman/shellcheck/wiki/SC2230) - _which is non-standard. Use builtin 'command -v' instead_

Reply:
Thing is these are actually not 100% equivalent (not sure that it matters for our case but...), I thought we'd disabled this lint globally.

```console
$ command -V command
command is a shell builtin
$ command -V which
which is hashed (/usr/bin/which)

$ which command
$ which which
/usr/bin/which

$ foo() {
> echo 'foo'
> }
$ which foo
$ command -v foo
foo
```

Reply:
I would probably use `command -v` if we were starting from scratch, but I'm not sure how worth it it is to change this everywhere 🤔

Reply:
Had no idea about that, but looks like its been brought up in the shellcheck project itself: [SC2230 - command -v is not a direct replacement for which](https://github.com/koalaman/shellcheck/issues/1162).

With that being the case I'd switch back to `which` and ignore it.

Reply:
yeah, in this case we use `bash` as the shell as well, so we're not trying to be strictly posix.

FWIW either should be fine for this case, but I think leaving it to `which` is fine, and previously set:
https://github.com/kubernetes/kubernetes/blob/139a13d312b1b11de7f3af63d9503365ff3a1e6a/hack/verify-shellcheck.sh#L34-L41

We should add a link to that issue where the lint is disabled.

Reply:
switched back to `which`